client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -26,36 +26,92 @@
     <div class="modal">
       <ion-header class="modal-header">
         <ion-title class="modal-header__title title-h2">
-          {{ getTitleAndSubtitle()[0] }}
+          {{ getTitleAndSubtitle().title }}
         </ion-title>
         <ion-text class="modal-header__text body">
-          {{ getTitleAndSubtitle()[1] }}
+          {{ getTitleAndSubtitle().subtitle }}
```

Comment:
You should probably add a `v-show="getTitleAndSubtitle().subtitle"`, else I'm afraid it's going to display `undefined`.